src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig

```diff
@@ -3,107 +3,116 @@
 {% form_theme form_content_location_add '@ibexadesign/ui/form/flat_widgets.html.twig' %}
 {% form_theme form_content_location_remove with ['@ibexadesign/ui/form/flat_widgets.html.twig', '@ibexadesign/ui/form_fields.html.twig'] %}
 
-<section>
+<section class="ibexa-content-locations">
     {{ form(form_content_location_add, {'action': path('ezplatform.location.add')}) }}
-    {% set body_rows = [] %}
-    {% if locations is not empty %}
-        {% for location in locations %}
-            {% set body_row_cols = [] %}
-
-            {% set col_raw %}
-                {{ form_widget(form_content_location_remove.locations[location.id], {'attr': {'disabled': not location.canDelete}}) }}
-            {% endset %}
-            {% set body_row_cols = body_row_cols|merge([{
-                has_checkbox: true,
-                content: col_raw,
-                raw: true,
-            }]) %}
-
-            {% set col_raw %}
-                {% include '@ibexadesign/ui/location_path.html.twig' with {'locations': location.pathLocations, 'link_last_element': true} %}
-            {% endset %}
-            {% set body_row_cols = body_row_cols|merge([{
-                content: col_raw,
-                raw: true,
-            }]) %}
-
-            {% set body_row_cols = body_row_cols|merge([
-                { content: location.childCount },
-            ]) %}
-
-            {% set col_raw %}
-                <label
-                    class="ibexa-checkbox-icon {{ not location.explicitlyHidden ? 'is-checked' }}{{ not can_hide[location.id] ? 'disabled' }}"
```

Comment:
Can we delete `ibexa-checkbox-icon` styles? I may be wrong, but it seems that it is only used here.